app/helpers/lit/frontend_helper.rb

```diff
@@ -5,11 +5,17 @@ module TranslationKeyWrapper
       def translate(key, options = {})
         options = options.with_indifferent_access
         key = scope_key_by_partial(key)
-        ret = super(key, options)
+
+        begin
+          content = super(key, options.merge(raise: true))
+        rescue I18n::MissingTranslationData
```

Comment:
I don't like this - it's changing the default behavior and changes content in helper, but maybe I'm missing the point why this has been added. Can you give me some additional explanation here?

Reply:
Hum, in retrospect I agree it's probably a bad idea.

I was to avoid the "translation missing" spans from rails.

Basically, `<%= t('woot') %>` becomes

``` html
<span class="lit-key-generic lit-key-highlight" data-key="woot" data-locale="fr" title="woot">
  <span class="translation_missing" title="translation missing: fr.woot">Woot</span>
</span>
```

Then when you edit that using the inline editing, the inline span is stored :disappointed:, and you have to edit it again and remove the span manually.

I could call `I18n.t()` instead of `t()`, but then we lose the lazy keys.